bugmon_tc/monitor/monitor.py

```diff
@@ -69,7 +71,7 @@ def fetch_bugs(self):
         response = self.bugsy.request("bug", params=QUERY)
         bugs = [EnhancedBug(self.bugsy, **bug) for bug in response["bugs"]]
         for bug in sorted(bugs, key=lambda bug: bug.id):
-            if self.is_actionable(bug):
+            if self.is_actionable(bug) is not None:
```

Comment:
Why `is not None`? This will always branch since `is_actionable` only returns bool values.

Reply:
Good catch.  This was leftover from another change I was working on.